Assets/MixedRealityToolkit.Tests/PlayModeTests/PointerTests.cs

```diff
@@ -101,15 +101,96 @@ public IEnumerator TestLinePointers()
         }
 
 
+        /// <summary>
+        /// Test pointers are correctly enabled when interacting with colliders that are visible, but whose
+        /// bounds are outside the camera FOV.
+        /// </summary>
+        [UnityTest]
+        public IEnumerator TestPointerFieldOfViewLargeCollider()
+        {
+            var rightHand = new TestHand(Handedness.Right);
+            yield return PlayModeTestUtilities.WaitForInputSystemUpdate();
+
+            var cube = GameObject.CreatePrimitive(PrimitiveType.Cube);
+            cube.AddComponent<NearInteractionGrabbable>();
+            cube.AddComponent<NearInteractionTouchableVolume>();
+            yield return rightHand.Show(Vector3.zero);
+            yield return PlayModeTestUtilities.WaitForInputSystemUpdate();
+
+            var spherePointer = PointerUtils.GetPointer<SpherePointer>(Handedness.Right);
+            var pokePointer = PointerUtils.GetPointer<PokePointer>(Handedness.Right);
+
+            yield return TestPointerFieldOfViewLargeColliderHelper(spherePointer, cube, rightHand);
+            yield return TestPointerFieldOfViewLargeColliderHelper(pokePointer, cube, rightHand);
+
+            rightHand.Hide();
+            GameObject.Destroy(cube);
+        }
+
+
+        private IEnumerator TestPointerFieldOfViewLargeColliderHelper(IMixedRealityPointer myPointer, GameObject cube, TestHand testHand)
+        {
+            cube.transform.localScale = new Vector3(3, 3, 0.05f);
+            float[] yOffsets = new float[] { -1f, 0f, 1f };
+            float[] xOffsets = new float[] { -1f, 0f, 1f };
+            float[] zOffsets = new float[] { 1f, -1f };
+
+            foreach (var zOffset in zOffsets)
+            {
+                foreach (var yOffset in yOffsets)
+                {
+                    foreach (var xOffset in xOffsets)
+                    {
+                        var cameraPos = CameraCache.Main.transform.position;
+                        var pos = new Vector3(cameraPos.x + xOffset, cameraPos.y + yOffset, cameraPos.z + zOffset);
+                        cube.transform.position = pos;
+                        yield return testHand.MoveTo(cube.transform.position);
+                        yield return PlayModeTestUtilities.WaitForInputSystemUpdate();
+                        bool isInFov = CameraCache.Main.IsInFOVCached(cube.GetComponent<BoxCollider>());
```

Comment:
nit: can getComponent of the box collider be pulled out of the for loops?

Reply:
Done